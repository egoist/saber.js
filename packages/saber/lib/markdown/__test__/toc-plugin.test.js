const Markdown = require('saber-markdown')
const fenceOptionsPlugin = require('../highlight-plugin')
const createEnv = require('./create-env')

test('main', () => {
  const md = new Markdown()
  const { env } = createEnv()
  md.use(fenceOptionsPlugin)
  const html = md.render(
    `
    # Fate
    ## Assassin
    ### ibn al-Sabbah
    ### Angra Mainyu
    ## Lancer
    ### Diarmuid Ua Duibhne
    ### CÃºchulainn
    `,
    env
  )
  expect(html).toMatchSnapshot()
})