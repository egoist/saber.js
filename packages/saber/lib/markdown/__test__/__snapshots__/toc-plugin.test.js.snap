// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main 1`] = `
"<pre><code># Fate
## Assassin
### ibn al-Sabbah
### Angra Mainyu
## Lancer
### Diarmuid Ua Duibhne
### CÃºchulainn
</code></pre>
"
`;
